ignore.js

```diff
@@ -1,5 +1,6 @@
 const ignoredFiles = [
   ".all-contributorsrc",
+  ".babelrc.js",
```

Comment:
consider migrate to babel.config.js

Reply:
I have investigated that and we gain nothing vs the `.babelrc.js` format so I'm gonna skip that for now.